src/components/common/Section.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+const Section = ({children}) => {
+  return (
+    <div className='section'>
+        {children}
+    </div>
+  )
+}
+
+export const SectionTitle = ({children}) => {
+    return (
+        <div className='title'>
+            {children}
+        </div>
+    )
+}
+
+export const SectionBody = ({children}) => {
+    return (
+        <div className='body'>
```

Comment:
Be carefull with class names, there are some methodoligies for css clean code like [BEM](http://getbem.com/introduction/)